app/components/LogIn/LogIn.js

```diff
@@ -15,7 +15,7 @@ class LogIn extends Component {
 
   signIn(e) {
     e.preventDefault()
```

Comment:
Why are you doing `e.preventDefault()` here? You will run into an error with this in your test because your mock event that you're simulating does not have a `preventDefault` method.